ProjectMarsLanguageFeature/Features/Language.feature

```diff
@@ -5,20 +5,20 @@ I want to add, delete and Edit language feature.
 So that the people seeking for the languages can look at what details I hold.
 
 @deleteLanguage
```

Comment:
you did not need to make two separate projects for skills and language, normally we put them in together in one project!

Reply:
I will take care of this in Mars Competition